platform/ui/tailwind.config.js

```diff
@@ -253,6 +245,7 @@ module.exports = {
     },
     fontFamily: {
       sans: [
+        'Lato',
```

Comment:
Is it corrent? maybe create font-lato.

Reply:
We're gonna be using `Lato` as default sans-serif font. That's why I've added `Lato` as a primary `sans` font. If in the future we want to change the default sans font, we can just replace `Lato` for something else, instead of having an additional CSS class for `font-lato`.

Reply:
Make sense